reference/faq/known-issues.md

```diff
@@ -67,6 +67,15 @@ you can while deploying do whatever you want (compile stuff or generate
 anything you need on the storage). But once deployed the main file-system
 will turn read-only.
 
+## Do you add custom HTTP headers?
+
+We add the following custom HTTP headers to let application knows the information about the connection.
+
+* "X-Forwarded-Proto": The protocol forwarded to the application, e.g. "http", "https".
+* "X-Client-IP": The remote IP address of the request.
+* "X-Client-SSL": Set "on" only if the client is using SSL connection.
+* "X-Original-Route": The route in `.platform/routes.yaml` which is used currently.
```

Comment:
This will be a templated URL, with {default} in it?  An example would be helpful here.